src/lib/common/data_getter.ts

```diff
@@ -0,0 +1,8 @@
+import type { FeatureCollection } from "geojson";
+import authoritiesUrl from "../../../assets/authorities.geojson?url";
+
+export async function getAuthoritiesData(): Promise<FeatureCollection> {
```

Comment:
Optional: don't need to say `Promise`, async implies it
"Async functions always return a promise. If the return value of an async function is not explicitly a promise, it will be implicitly wrapped in a promise. "
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

Reply:
I get red squiggles unless I put Promise, maybe an issue with my VSCode ts extension